johnsonandjohnson/Bodiless-JS
# feat(gatsby-theme-bodiless): pull upstream changes.

## Changes
Add the ability to pull upstream changes. 

## Test Instructions

  - Setup a working branch.
  - Push the branch remotely. 
   - Test pulling branch when:
    -- the branch is not pushed remotely.
    -- The branch is pushed remotely but there are no changes.
    -- There are changes that can be pulled on the remote branch ( accomplish this by editing and committing directly on github.
    -- The branch is pushed remotely and the chageset include changes to package-lock.json.
-More details see AC #303

## Related Issues

Fixed #303
